projects/aca-content/src/lib/components/search/search-results-row/search-results-row.component.ts

```diff
@@ -86,13 +94,42 @@ export class SearchResultsRowComponent implements OnInit, OnDestroy {
     this.node = this.context.row.node;
     this.isFile = this.node.entry.isFile;
 
-    const { name, properties } = this.node.entry;
-    const title = properties ? properties['cm:title'] : '';
-
+    const highlights: SearchEntryHighlight[] = this.node.entry['search']?.['highlight'];
+    let name = this.node.entry.name;
+    const properties = this.node.entry.properties;
+    let title = properties ? properties['cm:title'] : '';
```

Comment:
can be just: 
let title =  properties?.['cm:title'] || '';

Reply:
Fixed

Reply:
@MichalKinas small thing but you missed this for description too

Reply:
Ah thanks, will update it in next commit